terraform/main.tf

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+resource "juju_application" "upf" {
+  name  = var.app_name
+  model = var.model_name
+
+  charm {
+    name    = "sdcore-upf"
+    channel = var.channel
+  }
+  config = var.config
+  trust  = true
```

Comment:
Does this do anything with a machine charm? I think we can drop it otherwise.

Reply:
You're right, good catch, thanks